tests/js/plugin/enhancedanchoring-test.coffee

```diff
@@ -223,14 +228,13 @@ describe 'Annotator.Plugin.EnhancedAnchoring', ->
       assert.include hls, anchor2.highlight[anchor2.startPage]
       assert.notInclude hls, anchor3.highlight[anchor3.startPage]
 
-  describe 'Anchor.scrollIntoView()', ->
+  describe 'Anchor.scrollToView()', ->
     it 'calls scrollIntoView() on the highlight', ->
```

Comment:
> This isn't always the case, though, right? It should call it if it's real, otherwise it scrolls to the adjacent page and then the scrolling to the highlight happens asynchronously, correct?

Right. This whole section ([line 35-238](https://github.com/hypothesis/h/blob/add-highlight-test-and-change-scrolling/tests/js/plugin/enhancedanchoring-test.coffee#L35-L238)) is for testing simple, one-phase anchoring (and scrolling). The two-phase tests are [line 239-698](https://github.com/hypothesis/h/blob/add-highlight-test-and-change-scrolling/tests/js/plugin/enhancedanchoring-test.coffee#L239-L698).